packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -165,7 +165,7 @@ export const EventMeta = ({
                     event.lockTimeZoneToggleOnBookingPage ? "cursor-not-allowed" : ""
                   }`}>
                   <TimezoneSelect
-                    menuPosition="fixed"
+                    menuPosition="auto"
```

Comment:
Nah this is wrong, type `MenuPosition = "fixed" | "absolute"`

Reply:
@anikdhabal  Yeah, after making it absolute its working fine

Reply:
Updating the PR

Reply:
https://www.loom.com/share/4471b6345ad2488a9e7e27407af89745?sid=ef3c4a96-0b44-4e32-9a95-d547284bcf2f

Reply:
@hariombalhara @anikdhabal